integration-tests/src/test/java/oracle/weblogic/kubernetes/ItStickySession.java

```diff
@@ -278,6 +292,41 @@ public void testSameSessionStickinessUsingTraefik() {
     sendHttpRequestsToTestSessionStickinessAndVerify(hostName, ingressServiceNodePort);
   }
 
+  /**
+   * Verify that using OKD routes, two HTTP requests sent to WebLogic
+   * are directed to same WebLogic server.
+   * The test uses a web application deployed on WebLogic cluster to track HTTP session.
+   * server-affinity is achieved by Traefik ingress controller based on HTTP session information.
+   */
+  @Test
```

Comment:
This test should be skipped in non-OKD environment

Reply:
fixed